### PR TITLE
translate stage name for unprocessed order

### PR DESCRIPTION
**Pull request by shahad-yousif**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -113,9 +113,9 @@ en:
         pending: Pending
         paused: Paused
         not_published_yet: Not published yet
-        ad_run_out_of_tokens: Ad run out of Kidus
+        ad_run_out_of_tokens: The offer ran out of allocated Kidus
```
Comment (ssharba) [inline config/locales/en.yml]: The offer runs out of allocated Kidus

Comment (shahad-yousif) [reply config/locales/en.yml]: fixed it

File: config/locales/en.yml [hunk 2]
```diff
@@ -113,9 +113,9 @@ en:
         pending: Pending
         paused: Paused
         not_published_yet: Not published yet
-        ad_run_out_of_tokens: Ad run out of Kidus
+        ad_run_out_of_tokens: The offer runs out of allocated Kidus
         ad_paused_by_business: Paused by business
-        ad_is_expired: Ad is expired
+        ad_is_expired: Offer is expired
         publihed_to_customers: Published to customers
         ad_start_at: Ad start at
```
Comment (MwMaki) [inline config/locales/en.yml]: Why don't you replace all (Ad) word with (offer) ?

Comment (shahad-yousif) [reply config/locales/en.yml]: we need to replace only the content we may use it in other locations then will get missing translation

**Pull request merged**